pkdiagram/views/eventform.py

```
@@ -125,29 +129,137 @@ def nextItemInChain(item):
     #         itemClassName = ""
     #         parentName = ""
     #     _log.info(
-    #         f"AddAnythingDialog.onActiveFocusItemChanged: {parentName}.{itemName}[{itemClassName}]"
+    #         f"EventForm.onActiveFocusItemChanged: {parentName}.{itemName}[{itemClassName}]"
     #     )
 
-    def initForSelection(self, selection: list):
+    def addEvent(self, selection: list[Item] = None):
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)